### PR TITLE
Migrate SitewideLinks component to Psammead

### LINKED ISSUE #90: Migrate SitewideLinks Component to Psammead
**Is your feature request related to a problem? Please describe.**
We are creating a SitewideLinks component in https://github.com/BBC-News/simorgh/issues/1034, once this has been done we should migrate it to psammead 

Corresponding issue in Simorgh: https://github.com/BBC-News/simorgh/issues/1060

**Testing notes**
Storybook should still work, tests should still pass, and SiteWideLinks should be viewable in Storybook.

**Describe the solution you'd like**
Copy SitewideLinks into psammead as a new package

- [ ] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #90

Creates @bbc/psammead-sitewide-links from existing simorgh component

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-sitewide-links/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# psammead-sitewide-links &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-sitewide-links.svg)](https://www.npmjs.com/package/@bbc/psammead-sitewide-links) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+## Description
+
+The `@bbc/psammead-sitewide-links` package is designed to be used at the bottom of an article, however can be used anywhere on a page. These links are generally intended to be to more general and legal BBC pages, but do not have to be. It also includes information about copyright.
```
Comment (ChrisBAshton) [inline packages/components/psammead-sitewide-links/README.md:5]: Where is `externalLink` used? I've not actually looked at the component, so I'm reading this with fresh eyes. I understand the idea of links and a copyright message, but not sure where externalLink fits in.

File: packages/components/psammead-sitewide-links/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-sitewide-links/README.md",
+  "dependencies": {
+    "@bbc/gel-constants": "^0.1.1",
+    "@bbc/gel-foundations-styled-components": "^0.1.0",
+    "@bbc/psammead-styles": "^0.1.3",
+    "nanoid": "^2.0.0",
```
Comment (ChrisBAshton) [inline packages/components/psammead-sitewide-links/package.json:29]: As suggested in Slack, we shouldn't need `nanoid` and can use the `map(elm, index` if we want - but we can do that in a future PR. I appreciate this is a lift and shift.

**Review by ChrisBAshton — COMMENTED** [review]
Looks good, I would just elaborate on what the purpose of `externalLink` is.

File: packages/components/psammead-sitewide-links/README.md (hunk 1 above)
Comment (dr3) [reply packages/components/psammead-sitewide-links/README.md:5]: Its the link that goes after the copyright message 
image

Comment (ChrisBAshton) [reply packages/components/psammead-sitewide-links/README.md:5]: Therefore...

```suggestion
The `@bbc/psammead-sitewide-links` package is designed to be used in the footer of a generic BBC page. These links are generally intended for general and legal BBC pages, but do not have to be - and the component itself can be used anywhere in theory.

As well as the sitewide links, this component incorporates a copyright label, and a link next to it (which is intended to link to a disclaimer).
```

Actually, the more I look at it, the more I wonder why we've bundled sitewide links with copyright & external link 🤔 
screen shot 2018-12-07 at 15 39 03

Feels like this component should be simpler:

screen shot 2018-12-07 at 15 39 51

Copyright & external link should be tackled elsewhere (Simorgh's `FooterContainer`), IMO. I should have caught that sooner: https://github.com/BBC-News/simorgh/pull/1045/files#diff-40b843802de7eef375e3df8299b07701R50

Let's not hold up this PR - we can raise an issue to refactor it out in future.

File: packages/components/psammead-sitewide-links/package.json (hunk 2 above)
Comment (ChrisBAshton) [reply packages/components/psammead-sitewide-links/package.json:29]: And as pointed out by others, this is why we use nanoid! https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

File: .stylelintrc [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "rules": {
-    "declaration-no-important": true
+    "declaration-no-important": true,
+    "selector-type-no-unknown": [true, { "ignoreTypes": ["/^-styled-mixin/"] }]
```
Comment (sareh) [inline .stylelintrc]: Could you please explain why you added this rule?

Comment (ChrisBAshton) [reply .stylelintrc]: See https://github.com/BBC-News/psammead/pull/105#pullrequestreview-182717916

**Review by sareh — APPROVED** [review]
👍 Looks great!

**Review by ChrisBAshton — APPROVED** [review]

**Comment by dr3** [comment]
Make sure your on node `v8.11` and npm `v6.4.1`

Testing notes:
- Checkout this PR
- Run `npm install`
- Run `npm build`
- Run `npm run storybook`

You should see the SitewideLinks component in the storybook side column, It should look good at all widths

`npm run test` shouldnt fail.

File: packages/components/psammead-sitewide-links/README.md (hunk 1 above)
Comment (ChrisBAshton) [reply packages/components/psammead-sitewide-links/README.md:5]: Have raised that in issue #169, to look at next year.

**Comment by PriyaKR** [comment]
LGTM

**Pull request merged**